File: src/main/java/com/app/shoppingtally/admin/whatsNew/WhatsNew.java
package com.app.shoppingtally.admin.whatsNew;

import java.sql.Blob;

import com.app.shoppingtally.list.UserList;
import com.app.shoppingtally.user.User;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Lob;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WhatsNew {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	Long id;
	
	String title;
	String description;
	
	@Lob
	@Column(columnDefinition="LONGTEXT")
	String imageData;
}

File: employeeFrontend/src/models/WhatsNew.ts
export class WhatsNew{
    id:number;
    title:string;
    description:string;
    imageData:string;

    constructor(id,title,description,imageData){
        this.id = id;
        this.title = title;
        this.description = description;
        this.imageData = imageData
    }
}